eng/jacoco-test-coverage/pom.xml

```diff
@@ -159,17 +159,17 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-security-keyvault-certificates</artifactId>
-      <version>4.1.0</version> <!-- {x-version-update;com.azure:azure-security-keyvault-certificates;current} -->
+      <version>4.2.0-beta.1</version> <!-- {x-version-update;com.azure:azure-security-keyvault-certificates;current} -->
```

Comment:
Guessing you will be discarding these updates and creating a PR against master with these changes.
@vcolin7

Reply:
Yes, that's correct.